### ISSUE TITLE
New ctc decoder - np error (missing)

### ISSUE THREAD

**Issue by robmsmt** [mozilla/DeepSpeech#1724]
Exact same as config as 1722

```
D Starting batch...
D Finished batch step 253645.
100% (45 of 45) |###########################################################################################################################################################| Elapsed Time: 0:00:05 ETA:  00:00:00Traceback (most recent call last):
  File "DeepSpeech.py", line 615, in train
    job = coord.next_job(job)
  File "/fast/DeepSpeech-gpu/util/coordinator.py", line 562, in next_job
    return self.get_job(job.worker)
  File "/fast/DeepSpeech-gpu/util/coordinator.py", line 509, in get_job
    if self._next_epoch():
  File "/fast/DeepSpeech-gpu/util/coordinator.py", line 364, in _next_epoch
    mean_loss = np.mean(self._dev_losses[-FLAGS.earlystop_nsteps:-1])
NameError: name 'np' is not defined
Traceback (most recent call last):
  File "DeepSpeech.py", line 615, in train
    job = coord.next_job(job)
  File "/fast/DeepSpeech-gpu/util/coordinator.py", line 562, in next_job
    return self.get_job(job.worker)
  File "/fast/DeepSpeech-gpu/util/coordinator.py", line 509, in get_job
    if self._next_epoch():
  File "/fast/DeepSpeech-gpu/util/coordinator.py", line 364, in _next_epoch
    mean_loss = np.mean(self._dev_losses[-FLAGS.earlystop_nsteps:-1])
NameError: name 'np' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "DeepSpeech.py", line 940, in <module>
    tf.app.run(main)
  File "/fast/venv/py36-gpu/lib/python3.6/site-packages/tensorflow/python/platform/app.py", line 125, in run
    _sys.exit(main(argv))
  File "DeepSpeech.py", line 892, in main
    train()
  File "DeepSpeech.py", line 625, in train
    hook.end(session)
  File "/fast/venv/py36-gpu/lib/python3.6/site-packages/tensorflow/python/training/basic_session_run_hooks.py", line 588, in end
    self._save(session, last_step)
  File "/fast/venv/py36-gpu/lib/python3.6/site-packages/tensorflow/python/training/basic_session_run_hooks.py", line 599, in _save
    self._get_saver().save(session, self._save_path, global_step=step)
  File "/fast/venv/py36-gpu/lib/python3.6/site-packages/tensorflow/python/training/saver.py", line 1429, in save
    raise TypeError("'sess' must be a Session; %s" % sess)
TypeError: 'sess' must be a Session; <tensorflow.python.training.monitored_session.MonitoredSession object at 0x7f0778016208>
D Sending Job (ID: 3360, worker: 0, epoch: 318, set_name: dev)...
I Validation of Epoch 318 - loss: 29.742687
E name 'np' is not defined
D Closing queues...
D Queues closed.

```

presumably coordinator needs import numpy as np

**Comment by lissyx** [comment]
@robmsmt You can likely send a PR for that :-).

**Comment by robmsmt** [comment]
rerunning to see if that's all

**Comment by robmsmt** [comment]
Looks like Reuben fixed this with 0c6e695 I've tested and put a PR for the missing numpy.

**Comment by reuben** [comment]
Fixed in #1725.
